### ISSUE TITLE
期望增加显示编码格式功能

### ISSUE THREAD

**Issue by diqiu114** [cschlosser/doxdocgen#294]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by cschlosser** [comment]
Hi,
Could you rephrase your issue in English please?

**Comment by Autognosis** [comment]
He want to add a parameter to show the file is coded by UTF-8，GBK or etc. In China, there usually store Chinese characters by GBK or UTF-8. They can't be compatible with each other.
